pkg/telemetry/telemetry.go

```diff
@@ -28,15 +28,34 @@ var SiteID string
 var disabled bool
 var client piperhttp.Sender
 
+// SWA baseURL
+const baseURL = "https://webanalytics.cfapps.eu10.hana.ondemand.com"
+
+// SWA endpoint
+const endpoint = "/tracker/log"
+
+// Telemetry struct which holds necessary infos about telemetry
+type Telemetry struct {
+	baseData     BaseData
+	baseMetaData BaseMetaData
+	data         Data
+	provider     orchestrator.OrchestratorSpecificConfigProviding
+}
+
 // Initialize sets up the base telemetry data and is called in generated part of the steps
-func Initialize(telemetryDisabled bool, stepName string) {
-	disabled = telemetryDisabled
```

Comment:
this is checked later and cannot be removed

Reply:
see https://github.com/SAP/jenkins-library/pull/3213/files#diff-1daf3b94af9e46763e64ff02c31240f3c3b1421994d4d56e05769dfd4f765ba4R123